controllers/role.go

```diff
@@ -105,6 +105,24 @@ func (c *ApiController) UpdateRole() {
 		return
 	}
 
+	usersToCheck := role.Users
+
+	for _, userId := range usersToCheck {
+		user, err := object.GetUser(userId)
```

Comment:
don't make db query in loop. 
Fetch all required data before via GetUsersWithFilter or similar func

Reply:
done